lib/index.js

```diff
@@ -98,7 +98,17 @@ $("#hourly-forecast").click(function() {
   $(".hourly-forecast").toggle("slide");
 });
 
-
 $("#five-day-forecast").click(function() {
   $(".five-day-list").toggle("slide");
 });
+
+// this syntax makes the clock conststanly increment (still learning why)
+jQuery(function($) {
```

Comment:
I really like the implementation of a clock!

Reply:
This is great info thanks!